src/styles/components/pages/Signup/_Signup.scss

```
@@ -39,159 +33,231 @@
     }
   }
 
-  .page-content {
+  .column {
     padding: 0;
   }
 
   .columns {
     margin: 0;
     width: 100%;
-
-    .column {
-      display: flex;
-      flex-direction: column;
-      flex-shrink: inherit;
-      justify-content: center;
-      padding: 0;
-    }
   }
 
-  .has-no-padding {
-    padding: 0;
+  .confirmation-text {
+    margin-bottom: 12px;
```

on transforme tous les 12px en rem(12px)
```suggestion
    margin-bottom: rem(12px);
```

---

C'est fait partout